src/lib/hooks/use-best-quote/utils/issuance/legacy-quote.ts

```diff
@@ -0,0 +1,181 @@
+import {
```

Comment:
Added a new quote function as it requires quite some custom logic and wasn't sure I might be breaking other issuance things in swap or leverage